engine/tests/ElggCoreGetEntitiesFromAnnotationsTest.php

```diff
@@ -287,7 +288,6 @@ public function testElggGetEntitiesFromAnnotationsCalculateX() {
 						break;
 				}
 
-				$this->assertEqual($e->guid, $order[$i]);
```

Comment:
Not sure what this is supposed to test, but it did break the tests. Seems like order has changed somehow.